docs/02-for-app-authors/03-metainfo-guidelines/index.md

```diff
@@ -136,7 +136,7 @@ Please make sure to follow the [quality guidelines](/docs/for-app-authors/metain
 for `name` and `summary`.
 
 :::note
-The deprecated `developer_name` tag is also supported for backwards compatibility
+The deprecated `developer_name` tag is also supported for backwards compatibility. However, it's advisable to retain it to ensure compatibility with LTS software stores.
```

Comment:
"LTS" is not the proper qualifier.

"older releases / distributions" would be more accurate.